kin-sdk/kin-sdk-lib/build.gradle

```diff
@@ -24,6 +24,9 @@ android {
     buildTypes {
         debug {
             testCoverageEnabled true
+            buildConfigField "String", "INTEG_TESTS_NETWORK_FRIENDBOT", "\"$System.env.INTEG_TESTS_NETWORK_FRIENDBOT\""
```

Comment:
Just for your knowledge, fro example, System.getenv('VAR') returns null if not assigned. But If you use "$System.env.VAR" then it will return the string "null"

Reply:
thanks for the info, I think in this case that a real null is better is it's resulted with empty string, and not nothing at all (which result with compilation problem), same for null, as it will be odd to check if env is empty by comparing to null instead of empty string.

Reply:
totally right, thanks!